python/semantic_kernel/utils/settings.py

```diff
@@ -351,3 +351,27 @@ def booking_sample_settings_from_dot_env_as_dict() -> dict[str, str]:
     """
     client_id, tenant_id, client_secret = booking_sample_settings_from_dot_env()
     return {"client_id": client_id, "tenant_id": tenant_id, "client_secret": client_secret}
+
+
+def azure_container_apps_settings_from_dot_env() -> str:
+    """
+    Reads the Azure Container Apps environment variables from the .env file.
+    Returns:
+        str: Azure Container Apps pool management connection string
+    """
+    config = dotenv_values(".env")
+    connection_string = config.get("AZURE_CONTAINER_APPS_CONNECTION_STRING", None)
```

Comment:
We simply call this the pool management endpoint in the resource, UI, and in other libraries. Wonder if it'll be confusing if this is called a connection string here.

Reply:
Thank you, @anthonychu. I will update to `pool management endpoint` to help avoid confusion. Appreciate the feedback.